### ISSUE TITLE
release a version 

### ISSUE THREAD

**Issue by vamkaxifa** [mattn/go-oci8#368]
Hello , would you like to release a version, 'cause I have some trouble with go mod. When I build my project, I got error 
```
go: finding github.com/mattn/go-oci8 latest
build ProductRecordQuery: cannot load github.com/mattn/go-oci8: no matching versions for query "latest"
``` 

I'm thinking about if you can release a version, than I could  solve this out...

**Comment by MichaelS11** [comment]
@mattn What do you think?

**Comment by bwmarrin** [comment]
@vamitrou 
Does
```
go get github.com/mattn/go-oci8@master
```

Work for you?

**Comment by mattn** [comment]
I have plan to add go.mod in later.

**Comment by vamkaxifa** [comment]
go get github.com/mattn/go-oci8@master

works for me, thanks a lot ~~

**Comment by mattn** [comment]
Released v0.0.1
